File: schaakrob-server/src/main/kotlin/com/vdzon/java/ui/MainUI.java
package com.vdzon.java.ui;


import javax.swing.*;

public class MainUI {
  public static void main(String[] args) {

    SwingUtilities.invokeLater(new Runnable() {
      public void run() {
        MyPanel mainPanel = new com.vdzon.java.ui.MyPanel();
        mainPanel.fullscreen();
      }

    });
  }


}

File: schaakrob-server/src/main/kotlin/com/vdzon/java/ui/MainUI.kt
package com.vdzon.java.ui

import javax.swing.SwingUtilities

object MainUI {
    @JvmStatic
    fun main(args: Array<String>) {
        SwingUtilities.invokeLater {
            val mainPanel = MyPanel()
            mainPanel.fullscreen()
        }
    }
}
